src/wheatley.ts

```diff
@@ -257,7 +257,10 @@ export class Wheatley extends EventEmitter {
         }
 
         this.client.on("ready", async () => {
-            await this.fetch_guild_info();
+            if (!this.freestanding) {
```

Comment:
`fetch_guild_info` still has to run in freestanding in order to allow local development of components that rely on guild info. The intent was to have fetch_guild_info fetch errors be non-fatal, if that's not currently the case it's best to update it to make it the case.

Reply:
```none
[09.10.23 22:33:52] [log]   Logged in as wheatley-test#3347 (from: Client.<anonymous>:64)
[09.10.23 22:33:53] [error] DiscordAPIError[50001]: Missing Access
    at handleErrors (C:\Users\user\projects\wheatley\node_modules\@discordjs\rest\dist\index.js:687:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async SequentialHandler.runRequest (C:\Users\user\projects\wheatley\node_modules\@discordjs\rest\dist\index.js:1072:23)
    at async SequentialHandler.queueRequest (C:\Users\user\projects\wheatley\node_modules\@discordjs\rest\dist\index.js:913:14)
    at async _REST.request (C:\Users\user\projects\wheatley\node_modules\@discordjs\rest\dist\index.js:1218:22)
    at async ChannelManager.fetch (C:\Users\user\projects\wheatley\node_modules\discord.js\src\managers\ChannelManager.js:123:18)
    at async file:///C:/Users/user/projects/wheatley/build/src/wheatley.js:219:29
    at async Promise.all (index 8)
    at async Wheatley.fetch_guild_info (file:///C:/Users/user/projects/wheatley/build/src/wheatley.js:218:9)
    at async Client.<anonymous> (file:///C:/Users/user/projects/wheatley/build/src/wheatley.js:173:13) {
  requestBody: { files: undefined, json: undefined },
  rawError: { message: 'Missing Access', code: 50001 },
  code: 50001,
  status: 403,
  method: 'GET',
  url: 'https://discord.com/api/v10/channels/802541516655951892'
}
Trace
    at Function.error (file:///C:/Users/user/projects/wheatley/build/src/utils/debugging-and-logging.js:38:17)
    at Client.<anonymous> (file:///C:/Users/user/projects/wheatley/build/src/wheatley.js:163:15)
    at Client.emit (node:events:527:28)
    at emitUnhandledRejectionOrErr (node:events:384:10)
    at processTicksAndRejections (node:internal/process/task_queues:85:21)
```
I'm not sure why, but the error currently is fatal.

Reply:
Perhaps because this is an `async` function in a `.on(...)` function, which doesn't accept `async` callables according to the type annotations, so there's no proper error handling.

Reply:
It'd be best to wrap fetches in `fetch_guild_info` with try/catches